### ISSUE TITLE
LDS only on training data

### ISSUE THREAD

**Issue by siemdejong** [siemdejong/skinstression#56]
Currently, the LDS is calculated on training, validation and test data at once, but according to the paper it must only be computed on the training data, which is logical, as we don't want to have information leakage from the test set to the training set.

This might be hard to implement though, as the separation of training/test indices happens after data preparation. Might as well fix the fold indices in a file or something or switch up orders.
